.circleci/config.yml

```diff
@@ -6,16 +6,16 @@ jobs:
     steps:
       - checkout
 
-      - run:
-          name: install dependencies
-          command: make venv
-
-      - run:
-          name: 'Integration Tests'
-          no_output_timeout: 20m
-          command: |
-            export LOGGING_CONF_FILE=$(pwd)/sample_logging.conf
-            make integration_test
+#      - run:
+#          name: install dependencies
+#          command: make venv
+#
+#      - run:
+#          name: 'Integration Tests'
+#          no_output_timeout: 20m
+#          command: |
+#            export LOGGING_CONF_FILE=$(pwd)/sample_logging.conf
+#            make integration_test
```

Comment:
Why keeping these commented lines here if this change is not temporary?

Reply:
Ah forgot to remove the file, fixed in https://github.com/transferwise/pipelinewise-target-snowflake/pull/226/commits/57deaf3c01dc79f42b6c71bb159f757a3c982cae